doc/installation.rst

```diff
@@ -40,14 +40,15 @@ Windows or Mac OSX users
 ========================
 
 For Windows and/or Mac OSX user we highly recommend to install and use Anaconda
-for you Python3 installation. First install anaconda including python 3.5 or
+for your Python3 installation. First install Conda including python 3.7 or

```

Comment:
Here you write `python 3.7`, in the next line it is `3.8.` I think it would be better to always use the same version number.

Reply:
Thanks for checking! Fixed it now